experimental/packages/opentelemetry-instrumentation-http/src/utils.ts

```diff
@@ -402,6 +402,19 @@ export const getOutgoingRequestAttributesOnResponse = (
   return stableAttributes;
 };
 
+/**
+ * Returns outgoing request Metric attributes scoped to the response data
+ * @param {Attributes} spanAttributes the span attributes
+ */
+export const getOutgoingRequestMetricAttributesOnResponse = (
+  spanAttributes: Attributes
+): Attributes => {
+  return {
```

Comment:
getIncomingStableRequestMetricAttributesOnResponse is actually checking for undefined before returning the metrics, I believe metrics SDK should handle undefined values correctly but it should be good to check for consistency in this file.

Reply:
reasonable, updated to match.